api/serializers.py

```diff
@@ -170,6 +172,26 @@ def validate(self, attrs):
                     "extended for more than 6 months."
                 )
 
+            # no modification of minutes, start_date and carryover with locked shifts
+            if Shift.objects.filter(contract=self.instance, locked=True).exists():
+                if self.instance.initial_carryover_minutes != initial_carryover_minutes:
+                    raise serializers.ValidationError(
+                        "The Carryover of a contract with locked shifts "
```

Comment:
Hier fehlt das `_("...")` für die Internationalization.
In den folgenden Nachrichten ebenfalls.